src/index.js

```diff
@@ -416,6 +423,7 @@ export default class RNPickerSelect extends PureComponent {
                 >
                     {this.renderPickerItems()}
                 </Picker>
+                {Icon && this.renderIcon()}
```

Comment:
not a fan of this syntax personally -- would prefer:

`{Icon ? this.renderIcon() : null}`